server/android_notification_server.go

```diff
@@ -43,6 +43,7 @@ func (me *AndroidNotificationServer) SendNotification(msg *PushNotification) Pus
 			"override_icon_url": msg.OverrideIconUrl,
 			"from_webhook":      msg.FromWebhook,
 			"version":           msg.Version,
+			"id":                msg.Id,
```

Comment:
can we rename this to `notificationId` ? see https://github.com/wix/react-native-notifications/blob/master/docs/advancedIos.md

Reply:
sure